### PR TITLE
front: fix scroll bar macro mode

### LINKED ISSUE #8187: Macro-micro switch : weird scroll bar in the macro mode
### What happened?

In the microscropic mode, there is a vertical scroll bar which should not be here : the NGE part should just be a little bit smaller to fit the page 

image

### What did you expect to happen?

I expected to have no scrollbar

### How can we reproduce it (as minimally and precisely as possible)?

as above

### On which environments the bug occurs?

Recette (SNCF)

### On which browser the bug occurs?

Google Chrome

### OSRD version (top right corner `Account` button > `Informations`)

7faec4a

### PR DESCRIPTION
**Pull request by Uriel-Sautron**
close #8187

### CODE REVIEW

**Review by Yohh — APPROVED** [review]
LGTM tested

File: front/src/styles/scss/applications/operationalStudies/_scenario.scss [hunk 1]
```diff
@@ -32,7 +32,6 @@
       color: var(--white);
       border: unset;
       border-radius: 4px;
-      font-size: 1.5rem;
```
Comment (anisometropie) [inline front/src/styles/scss/applications/operationalStudies/_scenario.scss:35]: can’t use a overflow: hidden or change the height ?

Comment (Yohh) [reply front/src/styles/scss/applications/operationalStudies/_scenario.scss:35]: with the font-size:
image

without:
image

it really seems better without

**Review by kmer2016 — APPROVED** [review]
LGTM and tested !

File: front/src/styles/scss/applications/operationalStudies/_scenario.scss (hunk 1 above)
Comment (Uriel-Sautron) [reply front/src/styles/scss/applications/operationalStudies/_scenario.scss:35]: With osrd the icons are now the right size, no need for the font-size here.
overflow: hidden cuts the lower part of the macro component

**Review by anisometropie — APPROVED** [review]

**Pull request merged**